podium-gateway/src/main/webapp/app/shared/auth/principal.service.ts

```diff
@@ -26,31 +26,32 @@ export class Principal {
     authenticate (_identity) {
         this._identity = _identity;
         this.authenticated = _identity !== null;
+        this.authenticationState.next(this._identity);
     }
 
-    hasAnyAuthority (authorities) {
+    hasAnyAuthority (authorities: string[]): Promise<boolean> {
```

Comment:
It is required for chaining it in the directive [hasAnyAuth.directive.ts](https://github.com/thehyve/podium/blob/c685d26dba96dd3db674b2ee96b3d780621ea5dc/podium-gateway/src/main/webapp/app/shared/auth/has-any-authority.directive.ts)